chainerrl/agents/nsq.py

```diff
@@ -55,7 +55,7 @@ def __init__(self, q_function, optimizer,
         self.target_q_function = copy.deepcopy(q_function)
         self.q_function = copy.deepcopy(self.shared_q_function)
 
-        async.assert_params_not_shared(self.shared_q_function, self.q_function)
+        async_.assert_params_not_shared(self.shared_q_function, self.q_function)
```

Comment:
Could you break this line (>79 chars)?

Reply:
I'm sorry for my mistakes. I break this line in the new commit.